### ISSUE TITLE
New `events` fields on Verify search responses

### ISSUE THREAD

**Issue by lornajane** [Nexmo/api-specification#116]
We've shipped a change to the API without a corresponding docs update. When searcing for verifications by request ID(s), the response(s) will now include an `events` element showing which events have occurred.

Example response:
```
{
    "request_id": "2984392fa97f49e6a44f8eaec8746eef",
    "account_id": "[api key]",
    "number": "[phone number]",
    "sender_id": "verify",
    "date_submitted": "2019-05-06 08:57:04",
    "date_finalized": "",
    "checks": [],
    "first_event_date": "2019-05-06 08:57:04",
    "last_event_date": "2019-05-06 08:59:09",
    "price": "0.10000000",
    "currency": "EUR",
    "status": "IN PROGRESS",
    "events": [
        {
            "id": "1300000026F52C97",
            "type": "sms"
        },
        {
            "id": "e9d9efd56657cab826b84f4e5abc3b96",
            "type": "tts"
        }
    ]
}
```

**Comment by lornajane** [comment]
Some information on internal wiki here https://nexmoinc.atlassian.net/wiki/spaces/VER/blog/2019/04/02/1184268475/Verify+-+New+Pricing+Model#Search-API

**Comment by lornajane** [comment]
Was fixed by #123
